css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,128 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-border-radius-rounded: $border-radius-rounded;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	outline-color: $text;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	.segmented-control-previous,
+	.segmented-control-next {
+		padding: $sc-padding-sm;
+		border: $sc-border;
+		border-radius: $sc-border-radius-rounded;
+		background-color: $sc-active-background-color;
+		color: $sc-active-color;
+
+		&:hover {
+			border-color: $sc-active-hover-border-color;
+			background-color: $sc-active-hover-background-color;
+
+			@include forced-colors {
+				border-color: Highlight;
+				background-color: Highlight;
+				color: HighlightText;
+				forced-color-adjust: preserve-parent-color;
+			}
+		}
+	}
+
+	button {
```

Comment:
Are these styles a little opinionated to apply by default to all buttons within the `.segmented-controls` scope? Would it be worth moving these to their more specific selectors, and maybe letting case-by-case atomics/components do their thing?

Reply:
was more of an attempt to reduce duplication of properties.  I am happy to make them more specific again.